ruby/vampires.rb

```diff
@@ -1,4 +1,4 @@
-# Ken's Vamps
+## Ken's Vamps
 ########################   Loop of employees ##########################
 puts "How many Employees will be processed? (Please enter a digit.)"
 processed = gets.chomp.to_i

```

Comment:
On line 76 consider 

I think this should be your first check. If their name is Drac or Fang then we know we have a vampire. You can nest the rest of the checks under an else following this check. if vamp == true then no reason to check anything else.

The below is an example of handling the control flow in a more efficient computational pattern. Also, it is important to practice nesting conditions as many control flows will require you to do so.

```
if name == vamp
      vampire = true
else
      if other conditions
      elsif other conditions
      end
end
```